cloudinit/config/cc_mounts.py

```diff
@@ -245,20 +302,8 @@ def setup_swapfile(fname, size=None, maxsize=None):
         LOG.debug("Not creating swap: suggested size was 0")
         return
 
-    mbsize = str(int(size / (2 ** 20)))
-    msg = "creating swap file '%s' of %sMB" % (fname, mbsize)
-    try:
-        util.ensure_dir(tdir)
-        util.log_time(LOG.debug, msg, func=util.subp,
-                      args=[['sh', '-c',
-                             ('rm -f "$1" && umask 0066 && '
```

Comment:
Do we need to do something to replicate this umask call in the new codepaths?

Reply:
Actually this umask is wrong, it should be 177 (u=rw,go=) and it's handled on the last try/except of create_swapfile() function.